src/CommunityToolkit.Aspire.Hosting.Python.Extensions/UvAppResource.cs

```diff
@@ -6,9 +6,10 @@ namespace Aspire.Hosting.ApplicationModel;
 /// Represents a Uv application.
 /// </summary>
 /// <param name="name">The name of the resource.</param>
+/// <param name="executablePath">The path to the executable used to run the python app.</param>
 /// <param name="workingDirectory">The working directory for uv.</param>
-public class UvAppResource(string name, string workingDirectory)
-    : PythonAppResource(name, "uv", workingDirectory), IResourceWithServiceDiscovery
+public class UvAppResource(string name, string executablePath, string workingDirectory)
```

Comment:
This is a breaking change.

Reply:
It sure is.

The uv extensions are under the umbrella of being experimental (there's an `Experimental` attribute on them) warning that the API is in flux.

Reply:
Great.